File: app/src/main/java/one/plaza/nightwaveplaza/Api2/Models/Song.java
package one.plaza.nightwaveplaza.Api2.Models;

import com.google.gson.annotations.SerializedName;

public class Song {
    public String id;
    public String artist;
    public String title;
    public String album;
    public int position;
    public int length;
    @SerializedName("artwork_src") public String artworkSrc;
    public int reactions;
}

File: app/src/main/java/one/plaza/nightwaveplaza/api/Song.kt
package one.plaza.nightwaveplaza.api

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Song(
    val id: String = "",
    val artist: String = "",
    val album: String = "",
    val title: String = "",
    val length: Int = 0,
    @SerialName("artwork_src")
    val artworkSrc: String = "",
    @SerialName("artwork_sm_src")
    val artworkSmSrc: String = "",
    @SerialName("preview_src")
    val previewSrc: String = ""
)